plugin rename

guywillis opened the pull request:
resolves https://github.com/cgkineo/adapt-mediaWithBenefits/issues/5

* Rename plugin from Media with Benefits to Media Gallery

guywillis committed:
plugin rename

moloko commented on js/mediaGalleryView.js:
```diff
@@ -143,21 +143,24 @@ define([
       $buttonText.html(this.model.get('_transcript').inlineTranscriptCloseButton);
 
-      this.checkCompletionByTranscript();
+      if (this.model.get('_transcript')._setCompletionOnView !== false) {
+        this.setCompletionStatus();
+      }
```
was this change deliberate?

guywillis replied:
It is, to mirror the current media implementation of the inline transcript.

moloko commented on bower.json:
```diff
@@ -1,13 +1,13 @@
-  "displayName": "Media with benefits",
+  "homepage": "https://github.com/cgkineo/adapt-mediaGallery",
+  "issues": "https://github.com/cgkineo/adapt-mediaGallery/issues/ ",
+  "component": "mediaGallery",
```
not 100% sure about this but I _think_ this needs to be lowercase to match the component registration/name

moloko commented on bower.json:
```diff
@@ -1,13 +1,13 @@
-  "name": "adapt-mediaWithBenefits",
-  "version": "1.0.0",
+  "name": "adapt-mediaGallery",
+  "version": "2.0.0",
```
version v1.0.0 hasn't been released so there's no particular need to bump this to v2.0.0

guywillis committed:
version amend, component name amend

guywillis committed:
revert transcript change

moloko committed:
update for compatibility with the ES6 class version of adapt-contrib-media. and might as well convert to ES6 class while we're here...
bump FW version to 5.8 as that's the earliest version with proper support for spread syntax (with objects)

moloko committed:
honestly can't see why you'd ever want to show the transcript link as the label of the inline transcript button...

kirsty-hames reviewed (approved):
👀
